opennlp-tools/src/main/java/opennlp/tools/ml/maxent/GISTrainer.java

```diff
@@ -227,7 +227,7 @@ public GISModel trainModel(ObjectStream<Event> eventStream, int iterations, int
     Map<String, String> params = new HashMap<>();
     params.put(GIS.ITERATIONS_PARAM, Integer.toString(iterations));
     params.put(GIS.CUTOFF_PARAM, Integer.toString(cutoff));
-    indexer.init(params, new HashMap<>());
+    indexer.init(params);
```

Comment:
ok,  I wasn't sure why the indexer needed it.  I don't think it ever used it.  But I will put it back in.  If that is the case, it probably should not create a new Hashmap, but pass in a report map that someone else holds the reference.